src/main/java/org/myrobotlab/service/Polly.java

```diff
@@ -123,7 +123,7 @@ private AmazonPolly getPolly() {
     String secret = getKey(AMAZON_POLLY_USER_SECRET);
 
     if (key == null || secret == null) {
-      log.error("this service requires 2 keys - you can set them once with Security.setKeys(AMAZON_POLLY_USER_KEY, AMAZON_POLLY_USER_SECRET)");
+      error("this service requires 2 keys");

```

Comment:
technically speaking this service requires a key and a secret..

Reply:
heh ... sure .. Security service has a different concept of keys and values vs amazons concept of key and secret ...  
Anyway both are protected as "keys" in the Security service but I agree - its in the context of Polly so ... (changed it with correct verbiage)